SynthEHR.Core/Statistics/Distributions/IContinuousDistribution.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="IContinuousDistribution.cs" company="Math.NET">
+// Math.NET Numerics, part of the Math.NET Project
+// http://numerics.mathdotnet.com
+// http://github.com/mathnet/mathnet-numerics
+//
+// Copyright (c) 2009-2014 Math.NET
```

Comment:
I'm no expect, but do we have to up the copyright information on this file?

Reply:
No, it's just a copy of an MIT-licensed source file, no change required. It might be good to note a minor change in the `Normal.cs` file though (it has a couple of parabolic statistical functions we don't need expunged), I'll update for that.